designs/titan/config/index.js

```diff
@@ -71,6 +72,6 @@ export default {
     crotchSeamCurveAngle: { deg: 25, min: 0, max: 35 },
     waistBalance: { pct: 60, min: 30, max: 90 },
     grainlinePosition: { pct: 45, min: 30, max: 60 },
-    waistbandWidth: { pct: 3, min: 1, max: 6, snap: elastics, ...pctBasedOn('waistToFloor') },
+    waistbandWidth: { pct: 3, min: 0, max: 6, snap: elastics, ...pctBasedOn('waistToFloor') },
```

Comment:
This seems out of scope for this PR

Reply:
I thought it would be useful but I will change it back